package.json

```diff
@@ -13,28 +13,29 @@
     "lint-fix": "./node_modules/.bin/js-beautify -r index.js"
   },
   "dependencies": {
+    "MD5": "1.3.0",
     "cache-service": "1.3.5",
-    "cache-service-node-cache": "1.1.1",
+    "cache-service-node-cache": "2.0.0",
     "cache-service-redis": "2.0.0",
     "crypto-js": "3.1.2-2",
     "jsonwebtoken": "5.7.0",
-    "lodash": "3.10.1",
-    "uuid": "2.0.2",
-    "request": "^2.1.0",
-    "MD5": "^1.2.1"
+    "lodash": "4.17.11",
+    "node-cache": "4.2.0",
+    "request": "2.88.0",
+    "uuid": "2.0.2"
   },
   "devDependencies": {
     "debug": "2.6.8",
-    "js-beautify": "^1.8.9",
-    "jshint": "^2.10.1",
+    "js-beautify": "1.8.9",
+    "jshint": "2.10.1",
     "leche": "2.2.3",
     "mocha": "5.2.0",
     "nock": "7.2.2",
```

Comment:
Can't upgrade `nock` since it uses a later version of `debug`... which is not backwards compatible with node 4.